packages/devextreme/testing/testcafe/tests/scheduler/tooltipBehaviour/tooltipBehavior.ts

```diff
@@ -117,3 +119,76 @@ safeSizeTest('The tooltip should hide after manually scrolling in the browser',
     adaptivityEnabled,
   }));
 });
+
+safeSizeTest('Tooltip on mobile devices should have enough hight if there are async templates (React)', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const scheduler = new Scheduler('#container');
+
+  await t
+    .click(scheduler.collectors.find('7').element);
+
+  await ClientFunction(() => {
```

Comment:
Minor comment.

In my opinion - it's better to move the client functions separately.
It makes code more clear & obvious.
But it's only my opinion :)

E.g
```typescript
  const resolveWindowDeferred = ClientFunction(() => { (window as any).deferred.resolve(); });
  
  test('my test case', async (t) => {
  // other test code
  
  await resolveWindowDeferred ();
  
  // other test code
  );
```

Reply:
It's bad to move it to file scope, better to keep it in test scope.
I'll extract this to separate variable

Reply:
I usually move it to file scope because these functions can be used in many test cases and one function can be used in test case, before and after hooks :)

So, I don't think that it's bad.

Reply:
This very function is intended to be used only in this test